launch_ros/launch_ros/actions/node.py

```diff
@@ -340,16 +340,21 @@ def execute(self, context: LaunchContext) -> Optional[List[Action]]:
         self._perform_substitutions(context)
         # Prepare the ros_specific_arguments list and add it to the context so that the
         # LocalSubstitution placeholders added to the the cmd can be expanded using the contents.
-        ros_specific_arguments = []  # type: List[Text]
+        ros_specific_arguments = {}
```

Comment:
I'm pretty sure you'll need type annotations here. Please setup and use mypy to check your code when making changes. We'll likely add a unit test for mypy when this pr is finished: https://github.com/ament/ament_lint/pull/154

Reply:
Solved in https://github.com/ros2/launch_ros/pull/51/commits/230129438e2f74fa4390f5f34d3132134f775102